### DISCUSSION TITLE
Under which circumstance can ProcessModule.FileName be null?

### DISCUSSION THREAD

**Discussion by Symbai** [dotnet/runtime#63235] [Q&A] [answered]
It's marked as nullable string but I haven't seen any situation where a process module path is null/empty. The documentation page doesn't really give an answer.

**Comment by stephentoub** [comment, marked as answer]
I don't believe it can be null:
https://github.com/dotnet/runtime/pull/63272

**Reply by Symbai** [reply]
I believe the same, thank you
